src/transformers/configuration_bart.py

```diff
@@ -0,0 +1,92 @@
+# coding=utf-8
+# Copyright 2018 The Fairseq Authors and The HuggingFace Inc. team.
+# Copyright (c) 2018, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" BART configuration """
+
+
+import logging
+
+from .configuration_utils import PretrainedConfig
+
+
+logger = logging.getLogger(__name__)
+
+_bart_large_url = "https://s3.amazonaws.com/models.huggingface.co/bert/bart-large-config.json"
+BART_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "bart-large": _bart_large_url,
+    "bart-large-mnli": _bart_large_url,  # fine as same
```

Comment:
We've recently added an `architectures` field to `config.json` files, so that the correct model could be identified and displayed on the model pages, e.g. this [model finetuned on squad](https://huggingface.co/distilbert-base-cased-distilled-squad) recommends using `AutoModelForQuestionAnswering` instead of the standard `AutoModel`. See an example of this field on the [`bert-base-cased` configuration file](https://s3.amazonaws.com/models.huggingface.co/bert/bert-base-cased-config.json).

These two configurations would probably different were they mentioning which architectures they work for.